app/workers/import_user.rb

```diff
@@ -4,9 +4,9 @@ module Workers
   class ImportUser < ArchiveBase
     private
 
-    def perform_archive_job(user_id)
+    def perform_archive_job(user_id, import_parameters)
       user = User.find(user_id)
-      ImportService.new.import_by_user(user)
+      ImportService.new.import_by_user(user.username, import_parameters)
```

Comment:
So we have a user object, and then we get it's `username` just to look up the user again inside the `ImportService`. Maybe we can still pass the user object here?

Instead of changing this signature to username, maybe we could change `import_by_files` to accepting user object and in the rake task we could call `import_by_files` passing the `user` object that we find outside of the service?